CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 1.0.2 (7 December 2020)
```

Comment:
new feature => 1.1.0

Reply:
If we wanna be correct, it's breaking cause we change the API here, tho because we are the first consumers so far, I can safely assume that it's ok.